bond/util/validation.go

```diff
@@ -0,0 +1,70 @@
+package util
```

Comment:
Should this file be named `mac_validation.go` or will this file be used for other validation methods in the future?

Reply:
I hoped to get more of the validation code out of bond.go, for example the MTU one. Just did not want to polute this commit with unrelated changes.